packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -322,6 +323,11 @@ export class WorkflowWrapper {
 			return;
 		}
 
+		// check if the port value is unique, if so, we should not connect the incoming edge
+		if (targetNode.inputs.some((input) => input.unique && input.value?.[0] === sourceOutputPort.value?.[0])) {
+			return;
+		}
```

Comment:
A little weird as Yohann had pointed out, as the control would be better modelled at the operator level - this might also have problems with placeholder `nulls` from a general sense.

Reply:
Sure - I'll look into a way to do this at the operator level

Reply:
@YohannParis @mwdchang 
Okay, this should be at the operator level now. I've also added a check to see if the input value is `null` so we should be fine on that front as well.